release_dashboard/lib/widgets/create_release_substeps.dart

```diff
@@ -145,27 +210,29 @@ class InputAsSubstep extends StatelessWidget {
 class CheckboxListTileDropdown extends StatelessWidget {
   const CheckboxListTileDropdown({
     Key? key,
-    required this.index,
+    required this.substepName,
     required this.releaseData,
     required this.setReleaseData,
     required this.options,
+    required this.changeIsDropdownValid,
   }) : super(key: key);
 
-  final int index;
+  final String substepName;
   final Map<String, String?> releaseData;
   final SetReleaseData setReleaseData;
   final List<String> options;
+  final ChangeIsEachInputValid changeIsDropdownValid;
 
   @override
   Widget build(BuildContext context) {
     return Row(
       children: <Widget>[
         Text(
-          CreateReleaseSubsteps.substepTitles[index],
+          substepName,
           style: Theme.of(context).textTheme.subtitle1!.copyWith(color: Colors.grey[700]),
         ),
         // Only add a tooltip for the increment dropdown
-        if (index == 7)
+        if (substepName == CreateReleaseSubsteps.substepTitles[7])
```

Comment:
Avoid magic numbers. Try storing this as enum if it's required or having a more logical way to check for this (like a variable existing)

https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#avoid-mysterious-and-magical-numbers-that-lack-a-clear-derivation